exp-player/addon/styles/components/exp-overview.scss

```diff
@@ -1,5 +1,6 @@
 .format-average-label {
-    margin-left: 10px;
-    position: absolute;
+    display: block;
+    width: 100%;
+    margin-left: calc(50% + 10px);
```

Comment:
Can you comment on the meaning and origin of these numbers?

The big challenge we've always had for ISP is internationalization. How does it look in the three translations we have currently implemented (German, indonesian, and twice-as-long-a-word "x-pseudo"?